pkg/api/models/kluster_spec_audit.go

```diff
@@ -0,0 +1,52 @@
+// Code generated by go-swagger; DO NOT EDIT.
```

Comment:
I think this whole file is not required anymore. It seems like its from your attempt to include the audit config in the kluster spec, no?